commands.go

```diff
@@ -759,6 +759,10 @@ func cmdSsh(c *cli.Context) {
 			log.Fatalf("unable to get active host: %v", err)
 		}
 
+		if host == nil {
+			log.Fatalf("There is no active host. Please set it with docker-machine active <machine name>")
```

Comment:
Can you update it to use `c.App.Name` instead of hardcoding `docker-machine` please?